pkg/intermodule/adaptor/account.ts

```diff
@@ -0,0 +1,76 @@
+import { Result } from '@mikuroxina/mini-fn';
+import type { Account, AccountID } from '../../accounts/model/account.js';
+import type { FetchService } from '../../accounts/service/fetch.js';
+import type { FetchFollowService } from '../../accounts/service/fetchFollow.js';
+import type {
+  AccountModuleInterface,
+  PartialAccount,
+} from '../interfaces/account.js';
+
+export class AccountModule implements AccountModuleInterface {
+  constructor(
+    private readonly fetchService: FetchService,
+    private readonly fetchFollowService: FetchFollowService,
+  ) {}
+
+  async fetchAccount(id: AccountID): Promise<Result.Result<Error, Account>> {
+    const res = await this.fetchService.fetchAccountByID(id);
+    if (Result.isErr(res)) {
+      return res;
+    }
+
+    return res;
```

Comment:
```suggestion
    return res;
```
分岐に関わらず同じコードなので, err の分岐を書く必要はないかと.